src/components/collapsibleContainer/CollapsibleContainer.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import * as $ from 'jquery';
+import { LinkSvg } from '../svg/LinkSvg';
+import { Svg } from '../svg/Svg';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+import { IClassName } from '../../utils/ClassNameUtils';
+import { JSXRenderable } from '../../utils/JSXUtils';
+
+export interface ICollapsibleContainerOwnProps {
+  id: string;
+  title: JSXRenderable;
+  expandedOnMount?: boolean;
+  informationUrl?: string;
+  informationTooltip?: ITooltipProps;
+  className?: IClassName;
+  collapsibleHeaderClassName?: IClassName;
+  collapsibleBodyClassName?: IClassName;
+}
+
+export interface ICollapsibleContainerStateProps {
+  expanded?: boolean;
+}
+
+export interface ICollapsibleContainerDispatchProps {
+  onMount?: () => void;
+  onUnmount?: () => void;
+  onToggleExpandedState?: (currentExpandedState: boolean) => void;
+}
+
+export interface ICollapsibleContainerProps extends
+  ICollapsibleContainerOwnProps,
+  ICollapsibleContainerStateProps,
+  ICollapsibleContainerDispatchProps { }
+
+export class CollapsibleContainer extends React.Component<ICollapsibleContainerProps> {
+  private toggleCollapsibleBodyOnExpandedStateChange(nextProps: ICollapsibleContainerProps) {
+    const collapsibleBodySelector = `#${this.props.id} .collapsible-body`;
+    const animationOptions = { duration: 350 };
+
+    if (this.props.expanded && !nextProps.expanded) {
+      $(collapsibleBodySelector).slideUp(animationOptions);
+    } else if (!this.props.expanded && nextProps.expanded) {
+      $(collapsibleBodySelector).slideDown(animationOptions);
+    }
+  }
+
+  private getCollapsibleHeaderClass(): string {
+    return classNames(
+      'collapsible-header btn',
+      { active: this.props.expanded },
```

Comment:
I think `active: ...` should be on its own line just like `display-block` below, or the opposite.

Reply:
fixed